nengo_sphinx_theme/theme/404.html

```diff
@@ -0,0 +1,22 @@
+{% extends "layout.html" %}
+
+{% block body %}
+    <div class="row justify-content-center">
+      <div class="col-sm-12 col-lg-10 col-xl-9 text-center pt-5">
+        <h3 class="display-2">Error <strong>404</strong></h3>
+        <p class="display-4">
+          The page you are looking for cannot be found.
+        </p>
+        <p class="display-5">
+          If you just switched documentation versions, it is likely that the page you
+          were on doesn't exist in this version.
+        </p>
+        <form action="" method="get">
+        <input type="text" name="q" aria-labelledby="search-documentation" value="" />
```

Comment:
It would be nice to fill in the search value automatically using the requested page (from the URL that was originally requested -- you can get this in JS with `window.location`).

Reply:
Done